test/src/sdk-harness/test_projects/exponentiation/mod.rs

```diff
@@ -0,0 +1,103 @@
+use fuels::prelude::*;
+use fuels::tx::ContractId;
+use fuels_abigen_macro::abigen;
+use fuels::signers::LocalWallet;
+
+abigen!(
+    TestPowContract,
+    "test_artifacts/pow/out/debug/pow-abi.json"
+);
+
+#[tokio::test]
+#[should_panic(expected = "ArithmeticOverflow")]
+async fn overflowing_pow_u64_panics() {
+    let wallet = launch_provider_and_get_single_wallet().await;
+    let (pow_instance, _) = get_pow_test_instance(wallet).await;
+    pow_instance.u64_overflow(100u64, 100u64)
+        .call()
+        .await
+        .unwrap();
+}
+
+#[tokio::test]
+// TODO won't overflow until https://github.com/FuelLabs/fuel-specs/issues/90 lands
+// #[should_panic(expected = "ArithmeticOverflow")]
+async fn overflowing_pow_u32_panics() {
+    let wallet = launch_provider_and_get_single_wallet().await;
+    let (pow_instance, _) = get_pow_test_instance(wallet).await;
+    pow_instance.u32_overflow(10u32, 11u32)
+        .call()
+        .await
+        .unwrap();
+}
+
+#[tokio::test]
+#[should_panic(expected = "ArithmeticOverflow")]
+async fn overflowing_pow_u32_panics_max() {
+    let wallet = launch_provider_and_get_single_wallet().await;
+    let (pow_instance, _) = get_pow_test_instance(wallet).await;
+    pow_instance.u32_overflow(u32::MAX, u32::MAX)
+        .call()
+        .await
+        .unwrap();
+}
+
+#[tokio::test]
+// TODO won't overflow until https://github.com/FuelLabs/fuel-specs/issues/90 lands
```

Comment:
This can be removed.

Reply:
Keeping these tests around will allow us to catch regressions in expected behavior from the VM.

Reply:
I just mean the TODO can be removed.

Reply:
nvm.